src/Appveyor.TestLogger/AppveyorLogger.cs

```diff
@@ -24,14 +23,26 @@ class AppveyorLogger : ITestLogger
         /// </summary>
         public const string FriendlyName = "Appveyor";
 
-        /// <summary>
-        /// Initializes the Test Logger.
-        /// </summary>
-        /// <param name="events">Events that can be registered for.</param>
-        /// <param name="testRunDirectory">Test Run Directory</param>
-        public void Initialize(TestLoggerEvents events, string testRunDirectory)
+	    private AppveyorLoggerQueue queue;
```

Comment:
I had accidentally added a mess of tab characters and spaces.  Added `.editorconfig` to enforce standardization.